examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -696,20 +581,14 @@ extern "C" void app_main()
 
     chip::DeviceLayer::PlatformMgr().ScheduleWork(InitServer, reinterpret_cast<intptr_t>(nullptr));
 
-    std::string qrCodeText = createSetupPayload();
-    ESP_LOGI(TAG, "QR CODE Text: '%s'", qrCodeText.c_str());
-
-    {
-        std::vector<char> qrCode(3 * qrCodeText.size() + 1);
-        CHIP_ERROR error = EncodeQRCodeToUrl(qrCodeText.c_str(), qrCodeText.size(), qrCode.data(), qrCode.max_size());
-        if (error == CHIP_NO_ERROR)
-        {
-            ESP_LOGI(TAG, "Copy/paste the below URL in a browser to see the QR CODE:\n\t%s?data=%s", QRCODE_BASE_URL,
-                     qrCode.data());
-        }
-    }
+    // Print QR Code URL
+    PrintOnboardingCodes(chip::RendezvousInformationFlags(chip::RendezvousInformationFlag::kBLE));
```

Comment:
The previous code sets the flags based on the Rendezvous mode. Should we continue using that instead of hardcoding to BLE?